demos/phi-3-mini/llm.js

```diff
@@ -158,110 +149,35 @@ export class LLM {
     }
 
     async initialize_feed() {
-        // dispose of previous tensor
+        // dispose previous tensors
         for (const name in this.feed) {
             const t = this.feed[name];
-            if (t.location === "gpu-buffer") {
+            if (t.location === "gpu-buffer" || t.location === "ml-tensor") {
                 t.dispose();
```

Comment:
Does dispose indirectly call `t.mlTensor.destroy();`, or does `update_kv_cache()` now handle this?

Reply:
> Does dispose indirectly call t.mlTensor.destroy();

Yes.